src/index.js

```diff
@@ -2,6 +2,7 @@ import { MONGODB } from './constants';
 import client from './client';
 import mongoose from 'mongoose';
 import './client';
+import './bots/ama';
 import './bots/pins';
 import './bots/ping';
 import './bots/comp';
```

Comment:
not part of your pr but could you alphabetize the imports of the bots?

Reply:
sure will do